model-index.yml

```diff
@@ -59,4 +58,3 @@ Import:
   - configs/yolof/metafile.yml
   - configs/yolox/metafile.yml
   - configs/swin/metafile.yml
```

Comment:
We miss pvt and queryinst in the model-index. Can fix in this PR.

Reply:
[Fix] Add carafe, panoptic_fpn, queryinst, seesaw_loss in the model-index.